src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -415,15 +418,43 @@ public virtual AuthenticationProperties ConfigureExternalAuthenticationPropertie
         /// <summary>
         ///     Log SignInStatus for user and return SignInStatus
         /// </summary>
-        /// <param name="status"></param>
+        /// <param name="result"></param>
         /// <param name="user"></param>
         /// <param name="methodName"></param>
         /// <returns></returns>
-        protected async virtual Task<SignInResult> LogResultAsync(SignInResult status, TUser user, [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
+        protected virtual async Task<SignInResult> LogResultAsync(SignInResult result, TUser user,
+        [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
         {
-            status.Log(Logger, Resources.FormatLoggingResultMessage(methodName, await UserManager.GetUserIdAsync(user)));
+            if (Logger.IsEnabled(LogLevel.Information))
+            {
+                var status = "";
+
+                if (result.IsLockedOut)
```

Comment:
Consider moving this to a method in result that returns the right status string.